### DISCUSSION TITLE
Welcome to minigrace Discussions!

### DISCUSSION THREAD

**Discussion by apblack** [gracelang/minigrace#333] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the Grace community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kjx** [comment]
Hi, umm what's this?  sort of like issues without actual issues??

**Comment by apblack** [comment]
Exactly!  Its so that we can have a discussion, perhaps to decide if there should be an issue.  For example, I'm going to open a discussion on gracelang/language about **return**

**Reply by KimBruce** [reply]
Kim Bruce.  I teach with Grace and worry about static typing (among other things).
